bika/lims/browser/instrument.py

```diff
@@ -99,20 +112,22 @@ def folderitems(self):
         toshow = []
         for man in self.context.getMaintenanceTasks():
             toshow.append(man.UID())
-        for x in range (len(items)):
-            if not items[x].has_key('obj'): continue
-            obj = items[x]['obj']
+
+        for item in items:
+            if "obj" not in item:
+                continue
```

Comment:
This is not required, `bika_listing` already takes care of this: https://github.com/senaite/bika.lims/blob/master/bika/lims/browser/bika_listing.py#L1213